boards/common/stm32f103c8/dist/openocd.cfg

```diff
@@ -1,4 +1,25 @@
+source [find interface/stlink.cfg]
```

Comment:
I don't think this is required. This is because DEBUG_ADAPTER is set in Makefile.include. Then openocd make will automatically include `stlink.inc.mk`, which will add the right command.